### PR TITLE
Mgmt 2230 added admin users to cloud deployment

### PR DESCRIPTION
**Pull request by RazRegev**

### CODE REVIEW

**Comment by carbonin** [comment]
Needs to be added to the env list for the deployment as well, right?

**Comment by carbonin** [comment]
Also, should this be a secret or is this okay for the world to see?

File: openshift/template.yaml [hunk 1]
```diff
@@ -22,6 +22,9 @@ parameters:
 - name: ENABLE_AUTH
   value: ''
   required: true
+- name: ADMIN_USERS
+  value: 'ercohen,lgamliel,mfilanov,ygoldber'
```
[the remark this answers is not in the document]
Comment (ronniel1) [reply openshift/template.yaml:26]: also please change the commit to refer to MGMT-2627 and make NGNT-2230

[the remark this answers is not in the document]
Comment (ronniel1) [reply openshift/template.yaml:26]: Also need to change add this env variable at the section in the bottom of this file.  around line 276

**Comment by RazRegev** [comment]
will reopen after fixes

**Pull request closed without merging**